source/Calamari/Kubernetes/SetupKubectlAuthentication.cs

```diff
@@ -200,7 +232,14 @@ void ExecuteCommand(string executable, params string[] arguments)
 
         bool TryExecuteCommandWithVerboseLoggingOnly(params string[] arguments)
         {
-            return ExecuteCommandWithVerboseLoggingOnly(new CommandLineInvocation(kubectl.ExecutableLocation, arguments.Concat(new[] { "--request-timeout=1m" }).ToArray())).ExitCode == 0;
+            var result = kubectl.ExecuteCommandAndReturnOutput(
+                arguments.Concat(new[] { "--request-timeout=1m" }).ToArray());
+            foreach (var message in result.Output.Messages)
+            {
+                log.Verbose(message.Text);
+            }
+
+            return result.Result.ExitCode == 0;
```

Comment:
This is a genuinely nice clean up that we should do anyway It gets the same result as the whole method below as far as I can tell.